src/styles/core/_vars.scss

```diff
@@ -64,3 +64,7 @@ $total-columns: 12 !default;
 $card-horizontal-spacing: 10px;
 $card-horizontal-spacing-large: $card-horizontal-spacing * 2;
 $card-horizontal-spacing-small: $card-horizontal-spacing / 2;
+
+// Small vp padding and width
+$small-viewport-content-padding: 20px !default;
+$small-viewport-content-width: 100% !default;
```

Comment:
I'm not a CSS expert, so hopefully @dobromir-hristov and/or @marinaaisa can add some better advice on this:

I _think_ it might be slightly odd to have 100% width _and_ the 20px padding combined together (may run into issues depending on the `box-sizing`).

One possible alternative: don't set `$small-viewport-content-width` here—you can define it only when you need to adjust it and then change the code using it conditionally assign it if it's defined.

Another possible alternative: use `auto` instead of 100%. I think this would work but again I think someone else should comment on the best approach.

Reply:
This is why I originally wanted to set it to `unset`, but that's also not the best option. Maybe `auto` is a better approach? I set up some time to discuss this PR with Dobri tomorrow.